scapy/utils.py

```diff
@@ -1365,12 +1390,9 @@ def import_hexcap():
 
 
 @conf.commands.register
-def wireshark(pktlist, **kwargs):
+def wireshark(pktlist):
     """Run wireshark on a list of packets"""
-    f = get_temp_file()
-    wrpcap(f, pktlist, **kwargs)
-    with ContextManagerSubprocess("wireshark()", conf.prog.wireshark):
-        subprocess.Popen([conf.prog.wireshark, "-r", f])
+    tcpdump(pktlist, prog=conf.prog.wireshark)
```

Comment:
Note that it is always nice to be able to send a `linklayer` argument in `wireshark`, when passing raw data for instance.. that’s mostly what the kwargs were for

Would be nice if you add support for the `-y` argument by default

Reply:
Ah, you mean `linktype`?

Reply:
OK, I've added this in 6634a22.

Reply:
Yes sorry that's what I meant, thanks !